lib/measures/ConstrainSupplyAirTemperatureResetVerification/tests/test_measure.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+import openstudio
+from measure import ConstrainSupplyAirTemperatureResetVerification
+
+
+class TestConstrainSupplyAirTemperatureResetVerification:
+    def test_number_of_arguments_and_argument_names(self):
+        """
+        Test that the arguments are what we expect
+        """
+        # create an instance of the measure
+        measure = ConstrainSupplyAirTemperatureResetVerification()
+
+        # make an empty model
+        model = openstudio.model.Model()
+
+        # get arguments and test that they are expecting a failure
+        # because the model doesn't have a chiller
+        arguments = measure.arguments(model)
+
+        # Create dummy chiller object
+        air_loop = openstudio.model.AirLoopHVAC(model)
+        air_loop.setName("Test Air Loop")
+
+        # get arguments and test that they are what we are expecting
+        arguments = measure.arguments(model)
+        assert arguments.size() == 5
+        assert arguments[0].name() == "air_loop_name"
+        assert arguments[1].name() == "design_zone_cooling_air_temp"
+        assert arguments[2].name() == "idf_path"
+        assert arguments[3].name() == "output_dataset_path"
+        assert arguments[4].name() == "output_dir"
+
+    def test_good_argument_values(self):
+        """
+        Test running the measure with appropriate arguments, and that the
+        measure runs fine and with expected results
+        """
+
+        measure = ConstrainSupplyAirTemperatureResetVerification()
+
+        osw = openstudio.openstudioutilitiesfiletypes.WorkflowJSON()
+        runner = openstudio.measure.OSRunner(osw)
+
+        # make an empty model
+        model = openstudio.model.Model()
+
+        # Create dummy air loop object
+        air_loop = openstudio.model.AirLoopHVAC(model)
+        air_loop.setName("Test Air Loop")
+
+        # get arguments and test that they are what we are expecting
+        arguments = measure.arguments(model)
+        argument_map = openstudio.measure.convertOSArgumentVectorToMap(arguments)
+
+        args_dict = {
+            "air_loop_name": "Test Air Loop",
+            "design_zone_cooling_air_temp": 40,
+            "idf_path": "./lib/measures/ConstrainSupplyAirTemperatureResetVerification/tests/input/test.idf",
```

Comment:
If we don't need that argument we should probably remove the IDF file from this PR.